paddle/fluid/operators/sum_op.cc

```diff
@@ -36,8 +38,8 @@ class SumOp : public framework::OperatorWithKernel {
     }
 
     auto x_dims = ctx->GetInputsDim("X");
-    size_t N = x_dims.size();
-    PADDLE_ENFORCE_GT(N, 1, "Input tensors count should > 1.");
+    // size_t N = x_dims.size();
```

Comment:
Please delete these comments.

Reply:
add TODO here, maybe this check need to add back in the future.